packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/filters.dart

```diff
@@ -233,6 +233,26 @@ class Filter {
     Filter? filter8,
     Filter? filter9,
     Filter? filter10,
+    Filter? filter11,
+    Filter? filter12,
+    Filter? filter13,
+    Filter? filter14,
+    Filter? filter15,
+    Filter? filter16,
+    Filter? filter17,
+    Filter? filter18,
+    Filter? filter19,
+    Filter? filter20,
+    Filter? filter21,
+    Filter? filter22,
+    Filter? filter23,
+    Filter? filter24,
+    Filter? filter25,
+    Filter? filter26,
+    Filter? filter27,
+    Filter? filter28,
+    Filter? filter29,
+    Filter? filter30,
```

Comment:
Shouldn't these new parameters get forwarded to the ´_generateFilter(…)` call? Currently, they are ignored.

Same for `and(…)` below.

Also, the comment above still mentions 10 being the limit

Reply:
Thanks for the heads up 👍 . I've added to `_generateFilter()`.